programs/clearing_house/src/lib.rs

```diff
@@ -1749,7 +1723,7 @@ pub mod clearing_house {
     }
 
     #[access_control(
-        exchange_not_paused(&ctx.accounts.state)
+        withdraw_not_paused(&ctx.accounts.state)
     )]
     pub fn settle_expired_market(ctx: Context<UpdateAMM>, market_index: u16) -> Result<()> {
```

Comment:
why dont you want the admin to be able to settle a market when withdraws are paused?

Reply:
should this be admin only? right now its permissionless

Reply:
👍